features/step_definitions/common_steps.rb

```diff
@@ -1,6 +1,50 @@
 require 'uri'
 require 'securerandom'
 
+def choose_radio(label_or_radio, find_options={}, choose_options={})
+  if label_or_radio.is_a? Capybara::Node::Element and label_or_radio[:type] == 'radio'
+    radio = label_or_radio
+  else
+    radio = find_field(label_or_radio, find_options.merge({visible: :all}))
+  end
+
+  # If the label for this radio is not visible, it is effectively hidden from the user
+  radio.find_xpath('parent::label')[0].visible?.should be(true), "Expected label for radio button \"#{radio.value}\" to be visible"
```

Comment:
Do we always lay out our radio buttons inside their labels? I thought I'd also seen we sometimes do the `<label for=X>` thing, in which case this might not find it.